LayoutTests/fast/repaint/vertical-rl-box-shadow-repaint.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        .container {
+            margin: 20px;
+            border: 1px solid black;
+            width: 400px;
+            height: 250px;
+        }
+
+        .box {
+            margin: 25px;
+            width: 150px;
+            height: 100px;
+            background-color: green;
+            box-shadow: -10px -10px 0 10px red;
+        }
+        
+        body.changed .box {
+            box-shadow: -10px -10px 0 10px green;
+        }
+    </style>
+</head>
+<body>
+    <div class="container" style="writing-mode: vertical-rl">
+        <div class="box"></div>
+    </div>
+
+    <div class="container" style="writing-mode: horizontal-bt">
+        <div class="box"></div>
+    </div>
```

Comment:
i would love a test for sideways writing modes too if that's possible!

Reply:
Added.